template-html5-app/README.md

```diff
@@ -0,0 +1,251 @@
+# template-html5 - player-app
```

Comment:
I feel the readme here is too much to be included in this template/ example. A lot of them are common player usage that we probably can move to the root of the repo so that all templates/examples can share. 

Then within each template/example, it'll just include the steps required for the specific folder.

Reply:
I have slimmed the README down a bit

Reply:
Can we move the common player setup steps to the repo's readme as a standard player setup recommendation?

I still don't see anything moved to the common README file.